app/src/main/java/org/covidwatch/android/ui/reporting/VerifyPositiveDiagnosisFragment.kt

```diff
@@ -119,13 +121,13 @@ class VerifyPositiveDiagnosisFragment :
         val builder = MaterialDatePicker.Builder.datePicker()
         val constraints = CalendarConstraints.Builder()
 
-        val twoWeeksAgo = LocalDate.now().plusDays(-14).toInstant()
+        val twoWeeksAgo = ntpTime.nowAsLocalDate().plusDays(-14).toInstant()
```

Comment:
There is a problem with this solution.
Even if the dates that can be selected are correct the date picker shows the current month from the device's settings. 

I tried to use selection to current day when we open the dialog but it doesn't help.
@rasprague  Could you try to investigate if that somehow possible to fix with `MaterialDatePicker`?

Reply:
adding a setStart() to the date picker constraints fixes this